core/src/main/java/org/apache/iceberg/avro/TypeToSchema.java

```diff
@@ -214,4 +220,46 @@ public Schema primitive(Type.PrimitiveType primitive) {
 
     return primitiveSchema;
   }
+
+  private static String sanitize(String name) {
```

Comment:
Should this be in `AvroUtil` instead of `TypeToSchema`?

Reply:
I can move these methods, `validAvroName` and `sanitize` to AvroUtil